cwd/views/editor.pug

```diff
@@ -0,0 +1,32 @@
+doctype html
```

Comment:
Since you are using jQuery, consider using this file for templates for your various buttons.  For example, you can make an element with `display: none` in its style tag, and then instead of creating each element that you need entirely within JavaScript you can grab the template and then fill in whatever values are dynamic.  I think that could go a long way.

Reply:
This is now irrelevant because we are moving to React.